docs/builtins/Repl/typecheck.md

```diff
@@ -1,52 +1,54 @@
 ## typecheck
 
-Use `(typecheck "my-module")` to run the pact static typechecker on some module `my-module`
+Use `typecheck` to run the Pact static type checker on a specified `module` name.
 
 ### Basic syntax
 
-For some module `foo`:
+To run the static type checker on a specified `module` name, use the following syntax:
 
 ```pact
-(module foo GOV
-  (defcap GOV () true)
-
-  (defun plus-one (a) (+ a 1))
-)
-```
-
-To statically typecheck `foo`, call
-
-```pact
-(typecheck "foo")
+(typecheck module)
 ```
 
-If your module `foo` is under `(namespace "bar)`, then call `(typecheck "bar.foo")`
-
-## Arguments
+### Arguments
 
-Use the following arguments when calling `typecheck`:
+Use the following argument when calling the `typecheck` function:
 
 | Argument | Type | Description |
 |----------|------|-------------|
-| `module` | string | The module to run the static typechecker on |
+| `module` | string | Specifies the name of the module to run the static type checker on. |
 
-## Return value
+### Return value
 
-On typechecking succeess, `typecheck` returns the unit value `()`. Otherwise, it will throw an error.
+If type checking for the module is successful, the `typecheck` function returns the unit value `()`. 
+If type checking fails, the function returns an error.
```

Comment:
It throws an error. It doesn't return an error value, it will end program execution